### DISCUSSION TITLE
My License is locked

### DISCUSSION THREAD

**Discussion by EnnoKar** [aclap-dev/video-downloadhelper#1367] [License & Purchase requests]
### What's your issue?

After an update for Firefox my license is blocked. Unfortunately, I don't receive an email with the link to unlock as usual. Please help me.
email: enno14@hotmail.de

### The 10 first digits (only 10) of your license number or email address

fe923d71f0

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry, you should have received an email to reset your counter. Maybe in your spam box.
Anyway, I have reset your installation counter, should be OK now.

**Comment by EnnoKar** [comment]
Hi,

My spam box is empty. But thank you.

Mit freundlichen Grüßen
Enrico Karnatz
________________________________
